src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -499,22 +512,27 @@ const AutoCompleteRoot = styled(({isOpen, ...props}) => <div {...props} />)`
 `;
 
 const InputLoadingWrapper = styled(Flex)`
+  background: #fff;
   align-items: center;
-  border-bottom: 1px solid ${p => p.theme.borderLight};
   flex-shrink: 0;
   width: 30px;
 `;
 
+const StyledInputWrapper = styled(Flex)`
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  border-radius: ${p => `${p.theme.borderRadius} ${p.theme.borderRadius} 0 0`};
+  align-items: center;
+`;
+
 const StyledInput = styled(Input)`
   flex: 1;
+  border: 1px solid transparent;
 
   &,
   &:focus,
   &:active,
   &:hover {
-    border: 1px solid transparent;
-    border-bottom: 1px solid ${p => p.theme.borderLight};
-    border-radius: ${p => `${p.theme.borderRadius} ${p.theme.borderRadius} 0 0`};
+    border: 0;
```

Comment:
I don't totally understand what is going on here. we don't need any of this stuff anymore?

Reply:
I moved them to the parent flexbox (StyledInputWrapper) so the border spans the full width of the dropdown and not just the Input.